client/src/main/java/tds/assessment/Item.java

```diff
@@ -195,7 +195,7 @@ public String getItemFilePath() {
 
     public void setItemFilePath(String itemFilePath) {
         if (itemFilePath == null) {
-            itemFilePath = "";
+            throw new IllegalArgumentException("Item file path cannot be null");
```

Comment:
I think you can use itemFilePath = checkNotNull(itemFilepath);  In common legacy there is a Preconditions class.